components/Datasets/FileEditModal.vue

```diff
@@ -168,7 +172,7 @@
                   v-if="form.filetype === 'remote'"
                   class="fr-fieldset__element min-width-0"
                   :accordion="chooseTheCorrectLinkAccordionId"
-                  @blur="touch('url')"
+                  <BrandedButton @blur="touch('url')"
```

Comment:
I think there is something wrong with this line

Reply:
Weird… Fixed in 1d1e011